src/bin/anise-gui/main.rs

```diff
@@ -1,13 +1,14 @@
-use pretty_env_logger;
-use std::env::{set_var, var};
-
+#[allow(dead_code)]
```

Comment:
Where is the dead code? I don't immediately see it.

Reply:
It is the `LOG_VAR` variable, which becomes unused when compiling for a wasm target